### ISSUE TITLE
[Project] ML.NET 1.0 release

### ISSUE THREAD

**Issue by JRAlexander** [dotnet/docs#9126]
This is a meta-issue for the ML.NET 1.0 project.
Comment below to suggest new content, or updates to existing content.

Scenarios:
- [x] Binary classification
   - [x] [Sentiment Analysis](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis)
- [x] Multi-class classification
   - [x] [GitHub Issue Classification](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification)
- [x] Clustering
   - [x] [Iris Flower Clustering](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/iris-clustering)
- [x] Regression
   - [x] [Price Prediction](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/taxi-fare)
- [x] Recommendation 
   - [x] [Movie Recommendation](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommmendation)
- [x] Anomaly detection
   - [x] [Product Sales Analysis](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sales-anomaly-detection) 
- [x] Deep Learning
   - [x] Using TensorFlow with ML.NET
      - [x] [Image Classification](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification)

Features:
 - [x] Overview  
   - [x] What is ML.NET?
   - [x] Basic concepts for model training in ML.NET
- [ ] How-tos
   - [x] [Apply feature engineering for model training on categorical data.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-model-categorical-ml-net)
    - [x] [Apply feature engineering for model training on textual data with ML.NET.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-model-textual-ml-net)
   - [x] [Calculate metrics to evaluate model quality.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/verify-model-quality-ml-net)
   - [x] [Create a game match up list app with Infer.NET and probabilistic programming.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/matchup-app-infer-net)
   - [x] [Inspect intermediate data values during ML.NET pipeline processing.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/inspect-intermediate-data-ml-net)
   - [x] [Load data with many columns from a CSV file for machine learning processing.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-from-mult-column-csv-ml-net)
   - [x] [Load data from multiple files for machine learning processing.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-from-multiple-files-ml-net)
   - [x] [Load data from a text file for machine learning processing.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-from-text-file-ml-net)
   - [x] [Operationalize a trained machine learning model in apps.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/consuming-model-ml-net)
   - [x] [Train a machine learning model with data that's not in a text file.](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-non-file-training-data-ml-net)
   - [x] [Preprocess training data with normalizers to use in data processing.](normalizers-preprocess-data-ml-net)
   - [x] [Train a machine learning model using cross-validation.](train-cross-validation-ml-net)
   - [x] [Train a regression model to predict a value using ML.NET.](train-regression-model-ml-net)
   - [x] [Use the PredictionFunction to make one prediction at a time.](single-predict-model-ml-net)

**Comment by JRAlexander** [comment]
Closing As we have successfully launched ML.NET 1.0!
